src/sbuf.c

```diff
@@ -904,7 +915,7 @@ static bool setup_tls(struct tls_config *conf, const char *pfx, int sslmode,
 		err = tls_config_set_ecdhecurve(conf, ecdhecurve);
 		if (err) {
 			log_error("invalid %s_ecdhecurve: %s", pfx, ecdhecurve);
-			false;
+			return false;
```

Comment:
When rebasing there was this merge conflict, I left my version since it looks like a typo on the master version. I'm surprised this didn't trigger any compiler warnings.

Reply:
I saw this fix was included on master. Rebased this PR now to include it.

Reply:
Fixed separately: bfafe67ce084cea479826444fcadcf4771eac0b1

I also found another place with the same mistake. It is indeed mysterious that there was no compiler warning.